### PR TITLE
[PLAYER-5007] remove jquery, import underscore directly

### PR DESCRIPTION
**Pull request by ooyala-amohnatkin**

### CODE REVIEW

File: js/ad_manager_template.js [hunk 1]
```diff
@@ -5,7 +5,9 @@
  * version 0.1
  */
 
-OO.Ads.manager(function(_, $) {
+const { bind } = require('underscore');
```
Comment (nzaitcev) [inline js/ad_manager_template.js]: Is there an option to use built-in bind func or just arrow functions to get rid of underscore's bind?

Comment (ooyala-amohnatkin) [reply js/ad_manager_template.js]: Sure, but not in this PR

File: js/ooyala_ssai.js [hunk 2]
```diff
@@ -145,24 +159,24 @@ OO.Ads.manager(function(_, $)
       amc.willRequireEmbedCodeMetadata();
 
       // Add any player event listeners now
-      amc.addPlayerListener(amc.EVENTS.CONTENT_CHANGED, _.bind(_onContentChanged, this));
+      amc.addPlayerListener(amc.EVENTS.CONTENT_CHANGED, bind(_onContentChanged, this));
 
       // Stream URL
-      amc.addPlayerListener(amc.EVENTS.CONTENT_URL_CHANGED, _.bind(this.onContentUrlChanged, this));
-      amc.addPlayerListener(amc.EVENTS.PLAYHEAD_TIME_CHANGED , _.bind(this.onPlayheadTimeChanged, this));
+      amc.addPlayerListener(amc.EVENTS.CONTENT_URL_CHANGED, bind(this.onContentUrlChanged, this));
```
Comment (KseniiaSokolovaEpam) [inline js/ooyala_ssai.js]: Could we omit "bind" here?

File: js/ooyala_ssai.js [hunk 3]
```diff
@@ -1358,36 +1370,27 @@ OO.Ads.manager(function(_, $)
...
-                urls = _cacheBuster(urls);
-              }
-              OO.pixelPings(urls);
-              OO.log("Ooyala SSAI: \"" + trackingName + "\" tracking URLs pinged");
-            }
-            else
-            {
-              OO.log("Ooyala SSAI: No \"" + trackingName + "\" tracking URLs provided to ping");
-            }
+    var _pingTrackingUrls = bind(function(urlObject) {
```
Comment (KseniiaSokolovaEpam) [inline js/ooyala_ssai.js]: Would you like to change it (from var to const/let)?

Comment (ooyala-amohnatkin) [reply js/ooyala_ssai.js]: I think it's better to do this with adding eslint

File: js/ad_manager_template.js (hunk 1 above)
Comment (zelibobla) [reply js/ad_manager_template.js]: Why not in this PR? Ticket called "Remove Underscore and Jquery dependencies from html5-ad-plugins". I guess if underscore is still required, it should be explained.

File: js/ad_manager_vast.js [hunk 4]
```diff
@@ -407,18 +432,22 @@ OO.Ads.manager(function(_, $) {
...
+        if (vastRootElement.length === 0) {
+          _tryRaiseAdError("VAST: No VAST tags in XML");
+          return null;
+        }
+        if (vastRootElement.length > 1) {
+          _tryRaiseAdError("VAST: Multiple VAST tags in XML");
+          return null;
+        }
+        return vastRootElement[0];
+      } catch (error) {
```
Comment (zelibobla) [inline js/ad_manager_vast.js]: Throw a warning with the content of an error here, please. Suppressed exceptions are hard to debug.

File: js/ad_manager_vast.js [hunk 5]
```diff
@@ -517,8 +549,8 @@ OO.Ads.manager(function(_, $) {
      * @returns {boolean} true if the podded ads functionality is supported in the specified Vast version,
      *                    false otherwise
      */
-    var supportsPoddedAds = _.bind(function(version) {
-      return _.contains(SUPPORTED_FEATURES[getMajorVersion(version)], FEATURES.PODDED_ADS);
+    var supportsPoddedAds = bind(function(version) {
```
Comment (zelibobla) [inline js/ad_manager_vast.js]: Guess bind is redundant here

File: js/ad_manager_vast.js [hunk 6]
```diff
@@ -529,8 +561,8 @@ OO.Ads.manager(function(_, $) {
      * @returns {boolean} true if the ad fallback functionality is supported in the specified Vast version,
      *                    false otherwise
      */
-    var supportsAdFallback = _.bind(function(version) {
-      return _.contains(SUPPORTED_FEATURES[getMajorVersion(version)], FEATURES.AD_FALLBACK);
+    var supportsAdFallback = bind(function(version) {
```
Comment (zelibobla) [inline js/ad_manager_vast.js]: Guess bind is redundant here

File: js/ad_manager_vast.js [hunk 7]
```diff
@@ -539,7 +571,7 @@ OO.Ads.manager(function(_, $) {
      * @method Vast#getVastTemplate
      * @returns {object} The ad object that is formated to what we expect vast to look like.
      */
-    var getVastTemplate = _.bind(function() {
+    var getVastTemplate = bind(function() {
```
Comment (zelibobla) [inline js/ad_manager_vast.js]: Guess bind is redundant here

**Comment by ooyala-amohnatkin** [comment]
Now it should be ready, but keep in mind that there's a blocker https://jira.corp.ooyala.com/browse/PBS-1351 and the VAST plugin will not work until it's not fixed

**Pull request merged**